features/customers/login_error_messages.feature

```diff
@@ -0,0 +1,26 @@
+Feature: login errors should appear
+
+  As a customer who is trying to login
+  So that I can know why I couldn't login
+  I want to see error messages when login fails
+
+Background:
+
+  Given customer "Tom Foolery" has email "tom@foolery.com" and password "pass"
+  Given I am not logged in
+  When I visit the Login page
+
+Scenario: customer sees incorrect email if they type it in wrong
+  And I fill in "email" with "to@foolery.com"
```

Comment:
There is a easier way to do this. It's something like"As I login as xxx"

Reply:
I'm trying to test what happens if he tries to login with the wrong info, so i cant just do that